### PR TITLE
Added one more test for Input Fields Limit Negative Testing

### PR DESCRIPTION
**Pull request by JuliaEde**
Did some optimization, added Before Each and After Each, removed POMs from the top of the test file.

### CODE REVIEW

File: test/ui/specs/user/profileEditNegative.spec.js [hunk 1]
```diff
@@ -6,57 +6,63 @@ const {clearInputValue} = require('../../../helpers/uiMethods.helper');
 
 describe('Checking Input Fields Limit Negative Tests', () => {
 
-    it('First Name Max Length Validation (35 characters)', async () => {
+    beforeEach('Login with Existing User', async() => {
         await LoginPage.login(process.env.USER_EMAIL, process.env.USER_PASSWORD);
```
Comment (yuliahar) [inline test/ui/specs/user/profileEditNegative.spec.js:10]: Nice refactoring. Please move login to before('Login with Existing User', ... ) and rename beforeEach('Open user profile page', ...)

File: test/ui/specs/user/profileEditNegative.spec.js [hunk 2]
```diff
@@ -6,57 +6,63 @@ const {clearInputValue} = require('../../../helpers/uiMethods.helper');
...
         await EditProfilePage.btnSave.click();
         await expect(EditProfilePage.errorMessage).toHaveText('ValidationError: firstName: Max length is 35 characters, lastName: Max length is 35 characters');
-        await PublicationsPage.navBar.logout();
 
     });
 
-});
+    it('Should Show Error Message for All Fields', async () => {
+        await clearInputValue(await EditProfilePage.inputFirstName);
+        await EditProfilePage.inputFirstName.setValue('I wish you a Merry Christmas and a Happy New Year!');
```
Comment (yuliahar) [inline test/ui/specs/user/profileEditNegative.spec.js:51]: I like your test data :)

File: test/ui/specs/user/profileEditNegative.spec.js [hunk 3]
```diff
@@ -6,57 +6,63 @@ const {clearInputValue} = require('../../../helpers/uiMethods.helper');
...
+        await clearInputValue(await EditProfilePage.inputExternalLinkToProfileImage);
+        await EditProfilePage.inputExternalLinkToProfileImage.setValue("Oh the weather outside is frightful, But the fire is so delightful, And since we've no place to go, Let It Snow! Let It Snow! Let It Snow!It doesn't show signs of stopping, And I've bought some corn for popping, The lights are turned way down low, Let It!!!");
+        await clearInputValue(await EditProfilePage.inputAbout);
+        await EditProfilePage.inputAbout.setValue('"Oh the weather outside is frightful, But the fire is so delightful, And since weve no place to go, Let It Snow! Let It Snow! Let It Snow!It doesnttt show signs of stopping, And Ive bought some corn for popping, The lights are turned way down low, Let It!!!"');
+        await EditProfilePage.btnSave.click();
+        await expect(EditProfilePage.errorMessage).toHaveText('ValidationError: firstName: Max length is 35 characters, lastName: Max length is 35 characters, about: Max length is 255 characters, image: Max length is 255 characters, jobTitle: Max length is 255 characters');
+    });
+
+    afterEach('Logout', async() => {
```
Comment (yuliahar) [inline test/ui/specs/user/profileEditNegative.spec.js:64]: Please remove afterEach with logout. You don't need to logout every time after each it.

**Review by yuliahar — CHANGES_REQUESTED** [review]

File: test/ui/specs/user/profileEditNegative.spec.js (hunk 1 above)
Comment (ponomid) [reply test/ui/specs/user/profileEditNegative.spec.js:10]: @yuliahar Should we do empty line between our test steps and assertion? As i know it is a good practice.

File: test/ui/specs/user/profileEditNegative.spec.js (hunk 3 above)
Comment (JuliaEde) [reply test/ui/specs/user/profileEditNegative.spec.js:64]: Thank you for the feedback, I just tried to remove afterEach and it started to fail after the first it. I don't know what to do.

Comment (yuliahar) [reply test/ui/specs/user/profileEditNegative.spec.js]: Okay, leave afterEach without changes for now. I'll look into it later.

File: test/ui/specs/user/profileEditNegative.spec.js (hunk 1 above)
Comment (JuliaEde) [reply test/ui/specs/user/profileEditNegative.spec.js:10]: Yulia, I changed all that you have requested, removed login from beforeEach, added before with login and removed afterEach. But I don't know how to push it again to the same branch so I copied everything from webstorm file and then I edited the file directly on the Github website and pasted it. Please check.

Comment (JuliaEde) [reply test/ui/specs/user/profileEditNegative.spec.js:10]: It seems to be working

**Review by yuliahar — APPROVED** [review]

**Review by yuliahar — APPROVED** [review]
